content/applications/general/email_communication/email_domain.rst

```diff
@@ -83,81 +84,91 @@ and has not been altered during transport.
 - `OVH SPF <https://docs.ovh.com/us/en/domains/web_hosting_the_spf_record/>`_
 - `GoDaddy TXT record <https://www.godaddy.com/help/add-a-txt-record-19232>`_
 - `GoDaddy SPF <https://www.godaddy.com/help/add-an-spf-record-19218>`_
 - `GoDaddy DKIM <https://www.godaddy.com/help/add-a-cname-record-19236>`_
-- `NameCheap <https://www.namecheap.com/support/knowledgebase/article.aspx/317/2237/how-do-i-add-txtspfdkimdmarc-records-for-my-domain/>`_
+- `NameCheap <https://www.namecheap.com/support/knowledgebase/article.aspx/317/2237/how-do-i-add-
+  txtspfdkimdmarc-records-for-my-domain/>`_
 - `CloudFlare DNS <https://support.cloudflare.com/hc/en-us/articles/360019093151>`_
 - `Google Domains <https://support.google.com/domains/answer/3290350?hl=en>`_
 - `Azure DNS <https://docs.microsoft.com/en-us/azure/dns/dns-getstarted-portal>`_
 
-To fully test your configuration, the tool `Mail-Tester <https://www.mail-tester.com/>`_ will give 
-you a full overview of the content and configuration you have in one email sent! Mail-Tester can 
+To fully test your configuration, the tool `Mail-Tester <https://www.mail-tester.com/>`_ will give
+you a full overview of the content and configuration you have in one email sent! Mail-Tester can
 also be used for other lesser known providers.
 
 Use a default email address
 ===========================
 
-To force the email address from which emails are sent, you need to create the following key in
-the System Parameters of the database:
+To force the email address from which emails are sent, a combination of the following keys needs to
+be set in the system parameters of the database:
+
+- `mail.default.from`: accepts the local part or a complete email address as value
+- `mail.default.from_filter`: accepts a domain name or a full email address as value
+
+If the email address of the author does not match `mail.default.from_filter`, the email address is
+replaced by `mail.default.from` (if it contains a full email address) or a combination of
+`mail.default.from` and `mail.catchall.domain`.
 
-- If ``mail.default.from`` is set, and contains a full email address, all outgoing emails are sent
-  from the given address. This is a requirement to use `Outlook with Odoo 
-  <https://docs.microsoft.com/en-us/exchange/mail-flow-best-practices/how-to-set-up-a-multifunction-device-or-application-to-send-email-using-microsoft-365-or-office-365#option-1-authenticate-your-device-or-application-directly-with-a-microsoft-365-or-office-365-mailbox-and-send-mail-using-smtp-auth-client-submission>`_.
+For example `"Admin" <admin\@example.com>` => `"Admin" <notifications\@mycompany.odoo.com>`. If the
+`mail.default.from_filter` system parameter is not set then no email will be replaced; should only
+the domain be listed then all emails with that domain will be preserved; and if a full email
+address is listed then all emails will be replaced by the given address.
```

Comment:
> and if a full email address is listed then all emails will be replaced by the given address.

That's not 100% true, we never use the from_filter to replace the email (only the system parameters `mail.default.from` / `mail.catchall.domain`

This sentence is a bit confusing imo

It should be something like
> if the from_filter contains a full email address, and if the `mail.default.from` is the same as this address, then all email different than `mail.default.from` will be encapsulated in `mail.default.from`